howso/react_aggregate.amlg

```
@@ -0,0 +1,763 @@
+    ;               specified robust or non-robust type.
+    ; sample_model_fraction : optional, value 0.0 - 1.0, percent of model to use in sampling (using sampling without replacement).
+    ;               Applicable only to non-robust computation. Ignored if num_samples is specified.
+    ; sub_model_size: optional. if specified will calculate only on a sub model of the specified size from the full model.
+    ;               Applicable only to models > 1000 cases.
+    ; hyperparameter_param_path: optional. full path for hyperparameters to use for computation.
+    ;               If specified for any residual computations, takes precendence over action_feature parameter.
+    ; use_case_weights: optional, flag, if set to true will scale influence weights by each case's weight_feature weight
+    ; weight_feature: optional, default '.case_weight'.  name of feature whose values to use as case weights
+    ;               "generate_attempts" true or false. If true, outputs the total number of attempts to generate the unique case. Only applicable for generative
+    ;               reacts where generate_new_cases is "always" or "attempt". When used with ReactSeries, "series_generate_attempts" is also returned.
+    ; action_feature: optional, target feature for which to do computations. If "prediction_stats_action_feature" and "feature_influences_action_feature"
+	;				are not provided, they will default to this value.
+	; prediction_stats_action_feature: optional. When calculating residuals and prediction stats, uses this target features's hyperparameters. The trainee must
+	;				have been analyzed with this feature as the action feature first. If both "prediction_stats_action_feature" and "action_feature" are not provided,
+	;				by default residuals and prediction stats uses ".targetless" hyperparameters. If "action_feature" is provided, and this value is not provided, will
+	;				default to "action_feature".
```

I think this should be "will default to *the value of* 'action_feature'". 

This parameter will be of type: "string", so saying it defaults to "action_feature" can be a bit ambiguous/misleading